app/controllers/api/v0/order_cycles_controller.rb

```diff
@@ -80,8 +80,12 @@ def search_params
       end
 
       def permitted_ransack_params
-        [:name_or_meta_keywords_or_variants_display_as_or_variants_display_name_or_supplier_name_cont, # rubocop: disable Layout/LineLength
-         :with_properties, :primary_taxon_id_in_any]
+        [
+          "#{[:name, :meta_keywords, :variants_display_as,
+              :variants_display_name, :supplier_name]
+          .join(' or ')}_cont",
```

Comment:
Does this work? I thought it needed to be joined with `_or_`.

Reply:
The spec passes with either, so I'll just update it to `_or_` to be safe.

Reply:
I don't see that change. Is this covered by the specs? What happens if you intentionally add a typo? Does it still pass?

Reply:
It passes with an intentional typo. Should we write a spec for it?

Reply:
Yes. Good idea.

Reply:
Sorry I've been away for a month. I was trying to figure out how to write a spec for this and I'm stumped. The best I could do is just check the parameter against itself, which doesn't seem very robust.